user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -387,15 +394,15 @@
                 }
             </style>
 
-              <app-live-workshop 
-                title="Team Building!" 
-                descrip="You'll make friends here! Lots and lots of friends! Imagine being in a world, where all you see
-                are butterflies and rainbows and happiness. Waterfalls made out of sugar and dogs roaming the free 
+              <app-live-workshop
```

Comment:
There's so many of these hard coded in. Maybe worth doing something like this https://stackoverflow.com/questions/37520170/ngfor-repeating-components ? Also could just be unnecessary work

Reply:
Probably don't have time to fix these, but I agree that there are a few areas where I feel using ngFor would be beneficial

Reply:
Yeah we're def planning to change this after the hackathon once we get the image links in the db